tests/automation/message_checks.spec.ts

```diff
@@ -62,19 +62,20 @@ sessionTestTwoWindows('Send video 1:1', async ([windowA, windowB]) => {
     'tests/automation/fixtures/test-video.mp4',
   );
   await typeIntoInput(windowA, 'message-input-text-area', testMessage);
-  await sleepFor(100);
+  // give some time before we send the message, as the video preview takes some time to be added
+  await sleepFor(1000);
+
```

Comment:
maybe we can wait/loop for the preview to be rendered, so we can move faster and also make sure the preview logic is still working.